src/CMakeLists.txt

```diff
@@ -39,40 +70,6 @@ macro(CONFIG_INSTALL RELATIVE_PATH)
     endforeach(_SOURCE_FILE ${ARGN})
 endmacro()
 
-# Update __version__ and copy init.py to install directory
-configure_file(${PROJECT_SOURCE_DIR}/tomopy/__init__.py.in
-    ${PYTHON_SITE_DIR}/tomopy/__init__.py
-    @ONLY)
-
-# Copy over pure python module, python testing, and setup files
-file(GLOB _GENERAL "${PROJECT_SOURCE_DIR}/tomopy/*.py")
-file(GLOB _DATA    "${PROJECT_SOURCE_DIR}/tomopy/data/*.tif"
-                   "${PROJECT_SOURCE_DIR}/tomopy/data/*.h5")
-file(GLOB _MISC    "${PROJECT_SOURCE_DIR}/tomopy/misc/*.py")
-file(GLOB _PREP    "${PROJECT_SOURCE_DIR}/tomopy/prep/*.py")
-file(GLOB _RECON   "${PROJECT_SOURCE_DIR}/tomopy/recon/*.py")
-file(GLOB _SIM     "${PROJECT_SOURCE_DIR}/tomopy/sim/*.py")
-file(GLOB _UTIL    "${PROJECT_SOURCE_DIR}/tomopy/util/*.py")
-
-config_install("" ${_GENERAL})
-config_install("data"  ${_DATA})
-config_install("misc"  ${_MISC})
-config_install("prep"  ${_PREP})
-config_install("recon" ${_RECON})
-config_install("sim"   ${_SIM})
-config_install("util"  ${_UTIL})
-
-# copy of test directory to build directory
-execute_process(COMMAND ${CMAKE_COMMAND} -E copy_directory
-    ${PROJECT_SOURCE_DIR}/test ${PROJECT_BINARY_DIR}/test
-    WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
-
-# copy over setup.* files to build directory
-file(GLOB PYSETUP "${PROJECT_SOURCE_DIR}/setup.*")
-execute_process(COMMAND ${CMAKE_COMMAND} -E copy_if_different
-    ${PYSETUP} ${PROJECT_BINARY_DIR}/
-    WORKING_DIRECTORY ${PROJECT_BINARY_DIR})
-
```

Comment:
These copy operations on lines 42...76 are the ones which you want to keep for IDE compatibility correct? I can gate them with `if(NOT SKBUILD)`.

Do lines 89...92 also need remain unaltered for IDEs?